### PR TITLE
fix: Refine kuberneted docs

### PR DESCRIPTION
**Pull request by gaocegege**

### CODE REVIEW

**Review by VoVAllen — COMMENTED** [review]
We didn't use this. We used postgres image from docker now.

File: src/admin/kubernetes.md [hunk 1]
```diff
@@ -10,7 +10,9 @@ If you want to use `pgvecto.rs` in PostgreSQL which is running in Kubernetes, we
 | <a name="requirement_kubectl"></a> [kubectl](#requirement\_kubectl) | >= 1.14 |
 | <a name="requirement_kubernetes"></a> [kubernetes](#requirement\_kubernetes) | >= 1.24.0 |
 
-## Helm Install CloudNative-PG
+## Install CloudNative-PG
+
+We use [CloudNative-PG (cnpg)](https://cloudnative-pg.io/) to deploy PostgreSQL cluster in Kubernetes. You can install it by [helm](https://helm.sh/docs/intro/install/).
```
Comment (VoVAllen) [inline src/admin/kubernetes.md:15]: We didn't use this. We used postgres image from docker now.

Comment (gaocegege) [reply src/admin/kubernetes.md:15]: What do you mean?

cc @xieydd

**Review by xieydd — APPROVED** [review]
lgtm

**Review by VoVAllen — APPROVED** [review]

**Pull request merged**